main.cpp

```diff
@@ -73,29 +76,50 @@ struct <#structName1#>                                //4
 
 struct U
 {
-    float <#name1#> { 0 }, <#name2#> { 0 };
-    <#returnType#> <#memberFunction#>(<#type name#>* <#updatedValue#>)      //12
+    float oldFunk { 0 }, newFunk { 0 };
+    float regularFuncA(U* that, float* updatedFunk )      //12
     {
+        std::cout << "U's oldFunk value: " << that->oldFunk << std::endl;
```

Comment:
This function only had 1 parameter in the original placeholder code.
```
<#returnType#> <#memberFunction#>(<#type name#>* <#updatedValue#>)
```
- [ ] revise

Reply:
Ok, now the exercise makes sense.